tests/ui/type-alias-impl-trait/method_resolution.rs

```diff
@@ -0,0 +1,29 @@
+//! check that we do not unify `Bar<Foo>` with `BAr<u32>`, even though the
+//! `bar` method call can be resolved unambiguously by doing so.
```

Comment:
wait, why this? :thinking:

Reply:
I forgot to update the test descriptions in thr commit that affected them